capstone/src/main/webapp/index.jsp

```diff
@@ -1,5 +1,10 @@
 <html>
-<body>
-<h2>Hello World!</h2>
-</body>
+  <body>
+    <h2>Welcome to UniViz!</h2>
```

Comment:
also I think you [also want to add](https://google.github.io/styleguide/htmlcssguide.html#Encoding):
`<meta charset="utf-8">`
in a `head` element

Reply:
done!